### PR TITLE
Provided several fixes for app.betrybe.com

### PR DESCRIPTION
**Pull request by gralmeidan**
Some sections were inverted, too bright, and/or unreadable

### CODE REVIEW

**Comment by gralmeidan** [comment]
The page is only accessible to students so I grouped all the before/afters [here](https://imgur.com/a/yO8lSus)

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1134,6 +1134,39 @@ INVERT
...
+INVERT
+section.quiz-result__template summary
+section.quiz-result__template summary .block-code
+
+CSS
+.home-projects details,
+.header button,
+.header a,
+.module-index-content {
+   background-color: transparent
```
[the remark this answers is not in the document]
Comment (gralmeidan) [reply src/config/dynamic-theme-fixes.config:1148]: The original color is white, same as the background 
a 
INVERT was making it a much darker color 
a 
so I changed it to transparent 
a

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -1134,6 +1134,39 @@ INVERT
...
+.quiz-question label {
+   background-color: var(--darkreader-neutral-background)
+}
+.quiz-question label p {
+   color: rgb(180, 200, 255)
+}
+.c-bJifgQ:hover,
+.c-jPVfNN:checked + .c-bJifgQ {
+    background: initial !important
+}
```
[the remark this answers is not in the document]
Comment (gralmeidan) [reply src/config/dynamic-theme-fixes.config:1166]: Originally it overlayed the entire thing with this very bright color 
a
I'm honestly not so sure about the `initial` either but Invert didn't work and I'm not that good at raw css so this was the only way I found to make it not suck as much
a

I didn't know these selectors were auto-generated sorry, I'll be committing a fix

**Review by Gusted — CHANGES_REQUESTED** [review]
Please add `!important` behind the properties.

**Review by Gusted — APPROVED** [review]

**Pull request merged**